common/changes/791_CommandBarHCShift_2017-03-01-22-42.json

```diff
@@ -0,0 +1,20 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "adds pseudo:after class to highContrastBorder mixin, renders border on hover",
```

Comment:
This comment shows up in the release notes, so we usually start it with the component name and then a brief description of the functionality. E.g. "CommandBar: Improves accessibility by making borders visible in high contrast mode"